src/Illuminate/Foundation/Support/Providers/EventServiceProvider.php

```diff
@@ -25,7 +25,7 @@ class EventServiceProvider extends ServiceProvider
     /**
      * The model observers to register.
      *
-     * @var array
+     * @var array<string, array<int, string>>
```

Comment:
I can't recall, does Laravel use `class-string` anywhere else?

```php suggestion
     * @var array<class-string, class-string|array<int, class-string>>
```

Either way, observers can be of the format:
```php
$observers = [
    SomeClass::class => SomeClassObserver::class,
    // or
    SomeClass::class => [SomeClassObserver::class],
];